pkgs/top-level/all-packages.nix

```diff
@@ -23388,6 +23388,10 @@ with pkgs;
   openslp = callPackage ../development/libraries/openslp { };
 
   openstackclient = with python311Packages; toPythonApplication python-openstackclient;
+  openstackclient-full = openstackclient.overridePythonAttrs (attrs: {
+    dependencies =
+      attrs.dependencies ++ attrs.passthru.optional-dependencies.cli-plugins;
+  });
```

Comment:
```suggestion
  openstackclient-full = openstackclient.overridePythonAttrs (oldAttrs: {
    dependencies = oldAttrs.dependencies ++ oldAttrs.passthru.optional-dependencies.cli-plugins;
  });
```

Reply:
Thanks for the review! I fixed indentation and replaced attrs by oldAttrs